src/components/File/File.jsx

```diff
@@ -0,0 +1,124 @@
+/*
+Copyright (C) 2018 The Trustees of Indiana University
+SPDX-License-Identifier: BSD-3-Clause
+*/
+import classNames from "classnames";
+import * as PropTypes from "prop-types";
+import * as React from "react";
+import * as Rivet from "../util/Rivet";
+import Icon from "../util/RivetIcons";
+
+const classPrefix = "rvt-file";
+
+class File extends React.Component {
+  fileInput = null;
+
+  constructor(props) {
+    super(props);
+    this.fileInput = props.innerRef ? props.innerRef : React.createRef();
+  }
+
+  componentDidMount() {
+    const form = this.fileInput.current && this.fileInput.current.form;
+    if (form) {
+      form.onreset = (e) => {
+        this.forceUpdate();
+      };
+    }
+  }
```

Comment:
When beginning development for 2.x, we settled on not using lifecycle methods, and replacing them with hooks instead. Is it possible to use hooks here?

Reply:
Fixed. Thanks.